### PR TITLE
mentioning room members

### PR DESCRIPTION
**Pull request by sachintha-s**
**This PR includes mentioning room members and showing mentioned members.**

Untitled
#212

### CODE REVIEW

File: app/pubspec.yaml [hunk 1]
```diff
@@ -70,6 +70,8 @@ dependencies:
   get_time_ago: ^1.1.5
   implicitly_animated_reorderable_list: ^0.4.2
   cached_memory_image: ^1.3.1
+  flutter_mentions: ^2.0.1
+  flutter_html: ^3.0.0-alpha.5
```
Comment (gtalha07) [inline app/pubspec.yaml:74]: @gnunicorn @sachintha-s  Okay I've checked their compatibility issues with ```flutter_svg```. Apparently the dev is still using old version library of ```flutter_svg``` in the package which kinda sucks despite of many community requests. We can go with it but its alpha stage and many features missing and its not recommended for production. The workaround is to downgrade ```flutter_svg``` which also I would not recommend doing it.

Comment (gtalha07) [reply app/pubspec.yaml:74]: Just for heads-up as we will upgrade this soon as they add latest version of ```flutter_svg```.

**Comment by gtalha07** [comment]
Screenshot_1666010239
We need a bit of state handling here. It shows URL instead of markup text on sending message from effektio. But if I re-open the chat, it shows correctly.

**Comment by sachintha-s** [comment]
> Thanks for this @sachintha-s. But we would need this mentioned markup display in chat list item too. I suppose you planned to implement this there too?

I think it is already shown as it is in the Figma design.
image
What should be improved? Color?

**Comment by gtalha07** [comment]
> > Thanks for this @sachintha-s. But we would need this mentioned markup display in chat list item too. I suppose you planned to implement this there too?
> 
> I think it is already shown as it is in the Figma design. image What should be improved? Color?

Could be color with highlighted text OR something which is more prominent.

**Comment by sachintha-s** [comment]
> Also I would like mention display a bit more appealing. Is the mention widget customizable?

Yes, it is possible, if you have any design ideas please mention them.

**Comment by gnunicorn** [comment]
maybe @kokodavid has some ideas re design.

**Review by bitfriend — APPROVED** [review]
Looks good

File: app/pubspec.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gtalha07) [reply app/pubspec.yaml:74]: Already discussed [here](https://github.com/effektio/effektio/pull/277#discussion_r996994677)

**Comment by kokodavid** [comment]
Let me look into how we can do some designs on that.

**Comment by gtalha07** [comment]
> @gtalha07 should we be merging this as is for now and put up tickets for the remaining things, like a prettier design, to do for later? What do you think?

Yes, @sachintha-s will get this merged after applying review fixes.

**Pull request merged**